roles/pulp/defaults/main.yaml

```
@@ -1,10 +1,18 @@
 ---
-pulp_image: quay.io/pulp/pulp:stable
-pulp_ports:
-  - "8080:80"
+pulp_api_image: quay.io/pulp/pulp-minimal:stable
+pulp_content_image: quay.io/pulp/pulp-minimal:stable
+pulp_worker_image: quay.io/pulp/pulp-minimal:stable
+
+pulp_api_ports:
+  - "24817:80"
+pulp_content_ports:
+  - "24816:80"
+pulp_worker_count: 2
+
 pulp_volumes:
-  - /var/lib/pulp/settings:/etc/pulp:Z
-  - /var/lib/pulp/pulp_storage:/var/lib/pulp:Z
-  - /var/lib/pulp/pgsql:/var/lib/pgsql:Z
-  - /var/lib/pulp/containers:/var/lib/containers:Z
-pulp_container_name: pulp
+  - /var/lib/pulp/pulp_storage:/var/lib/pulp
+  - /var/lib/pulp/containers:/var/lib/containers
```

I don't think the containers mount is needed anymore with minimal containers and we only really need storage.

At that point, should we just expose `/var/lib/pulp` as `/var/lib/pulp`? And I still think you need the SELinux label.
```suggestion
  - /var/lib/pulp:/var/lib/pulp:Z
```

---

> And I still think you need the SELinux label.

I wish. It didn't work with :Z in my tests :/
That's we set `label=disable` on the containers.

---

And yes, you're right, we don't need the containers mount, see https://discourse.pulpproject.org/t/what-is-var-lib-containers-used-for-when-deploing-oci/1782